jest.config.js

```diff
@@ -23,10 +23,10 @@ module.exports = {
     "packages/remix-server-runtime",
     "packages/remix-testing",
   ],
+  reporters: [["github-actions", { silent: false }], "summary"],
```

Comment:
note that you might want something like `reporters: process.env.GITHUB_ACTIONS == null ? undefined : [['github-actions', {silent: false}], 'summary'],`, otherwise you get no output when running tests locally. This is probably something we should fix in Jest...